src/vim.rs

```diff
@@ -66,10 +57,6 @@ impl Vim {
         self.rpcclient.notify("cursor", json!([lnum, col]))
     }
 
-    pub fn setline(&self, lnum: u64, text: &[String]) -> Fallible<()> {
```

Comment:
This is unused right now, but I think it is very likely to be used in the short near future.

If the warning is disturbing you, please add `[allow(unused)]` to turn if off instead.